tutu-backend/src/main/scala/fi/oph/tutu/backend/domain/Hakemus.scala

```diff
@@ -113,10 +144,11 @@ case class PartialHakemus(
   paatosPvm: Option[LocalDateTime] = None,
   esittelijaOid: Option[String] = None,
   kasittelyVaihe: Option[KasittelyVaihe] = None,
-  pyydettavatAsiakirjat: Option[Seq[PyydettavaAsiakirja]] = None,
+  pyydettavatAsiakirjat: Seq[PyydettavaAsiakirja] = Seq.empty,
```

Comment:
Olisko kuitenkin selkeämpää jos PartialHakemuksen puolella olisi edelleen Option:in takana..

Reply:
Joo, olis. Muutetaan